notebooks/official/vector_search/sdk_vector_search_for_indexing.ipynb

```diff
@@ -491,7 +491,7 @@
         "                \"restricts\": [\n",
         "                    {\n",
         "                        \"namespace\": \"class\",\n",
-        "                        \"allow_list\": [\"even\" if index % 2 == 0 else \"odd\"],\n",
+        "                        \"allow\": [\"even\" if index % 2 == 0 else \"odd\"],\n",
```

Comment:
Based on the official documentation, we should use `allow` instead of `allow_list` in source file of vector index.


> - An optional restricts field can be included that specifies an array of TokenNamespace objects in restricts. For each object:
>    - Specify a namespace field that is the TokenNamespace.namespace.
>    - An optional `allow` field can be set to an array of strings which are the list of TokenNamespace.string_tokens.

ref: https://cloud.google.com/vertex-ai/docs/vector-search/setup/format-structure?hl=ja#data-file-formats